src/main/java/org/candlepin/async/tasks/ImportJob.java

```diff
@@ -230,7 +230,7 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
                 storedFileId, overrides, uploadedFileName);
 
             log.info("Async import complete.");
-            context.setJobResult(importRecord);
+            context.setJobResult(importRecord.toString());
```

Comment:
This was causing a de-serialization issue with the AsyncJobStatusDTO object because the "resultData" field would be an object instead of a string when you fetched the status of the ImportJob using the /jobs/{job_id} endpoint.

Reply:
nack on this

The way this is intended to work is that any job result is JSON serialized. There's an argument to be made as to whether or not we should be serializing model objects (even fake ones), but changing this to .toString completely changes the output of this job.

If there's a problem on the spec test side of things, that should be updated instead.

Reply:
That makes sense. I used the Request api to avoid the serialization issues!